controllers/telegram_handlers.py

```diff
@@ -299,7 +317,9 @@ async def get_faculty_feedback(
     user_response_args = user_response.split(" ")
     logger.info("Received input for faculty feedback")
 
-    if user_response_args == ["cancel"]:
+    cancelled = helpers.check_cancel(user_response_args)
```

Comment:
I would say don't make a helper function for such a simple thing. To know what is happening here we would have to check another file, which I don't like.